### DISCUSSION TITLE
Can connect to M5 Atom, but no devices seen

### DISCUSSION THREAD

**Discussion by jasonb194** [omerfaruk-aran/esphome_samsung_hvac_bus#50] [General]
Hello,

I have 2021 Samsung Mini-splits/heat pumps. I followed this guide and i can connect without issue, but the device is not seeing any devices. Note that I didn't know if my device is NASA or non-NASA, so I didn't define any devices to start with.

I also used the 12v feed from this post: https://github.com/lanwin/esphome_samsung_ac/discussions/39

This is the first time I have tried to set up an ESP32 device, does an initial device need to be set?

Also, in the instructions it says to boot up, i turned off power to the one indoor mini-split it is connected to, but the outdoor unit still has power. Do I need to power cycle the outdoor unit?

I am also seeing the following often in logs:

```
[16:18:30][W][samsung_ac:085]: Last transmission too long ago. Reset RX index.
[16:18:31][W][samsung_ac:020]: update
[16:18:31][C][samsung_ac:041]: Configured devices: 
[16:18:31][C][samsung_ac:061]: Discovered devices:
[16:18:31][C][samsung_ac:062]:   Outdoor: -
[16:18:31][C][samsung_ac:063]:   Indoor:  -
```

**Comment by lanwin** [comment]
Hi jasonb194, you dont need to do anything with your climate devices. If you connected them right and the cables are connected, they should appear under discovered devices (als long as they have power).

First it would be helpful when you post your yaml, so we can have a look.

Usually the following could be checked.
* Check your wiring (I had a lot problems cause the wire connection was loose) 
* Check if you really connected with the wiring to the outdoor unit not there wiring to wall control
* Test if swapping F1/F2 helps
* Check settings baud_rate from 9600 to 2400 (some older hardware uses a lower baud rate)
* Add that to your yaml witch dumps all data witch is received via RS484 to logs
```
  debug:
    direction: BOTH
    dummy_receiver: false
      after:
        delimiter: "\n"
        sequence:
        - lambda: UARTDebug::log_hex(direction, bytes, ' ');
```

**Comment by jasonb194** [comment]
Thank you for your help. As soon as I switched F1 and F2, data started streaming in. Thank you so much for this project. Finally off using the cloud for my mini-splits!

**Reply by jasonb194** [reply]
Oddly Enough, the other set of Minisplits I have had the wiring the other way around (correct based on instructions). I assume that the units can handle the messages in either configuration.

**Reply by lanwin** [reply]
Interesting. The most RS485 adapters I used always receive data when swapped but the data is not in a readable format. 

I added that to the Troubleshooting section of the docs.
